hazelcast/util.py

```diff
@@ -333,3 +382,79 @@ class IterationType:
 
     ENTRY = 2
     """Iterate over entries"""
+
+
+class UUIDUtil(object):
+    @staticmethod
+    def to_bits(value):
+        i = value.int
+        most_significant_bits = to_signed(i >> UUID_MSB_SHIFT, 64)
+        least_significant_bits = to_signed(i & UUID_LSB_MASK, 64)
+        return most_significant_bits, least_significant_bits
+
+    @staticmethod
+    def from_bits(most_significant_bits, least_significant_bits):
+        return uuid.UUID(
+            int=(
+                ((most_significant_bits << UUID_MSB_SHIFT) & UUID_MSB_MASK)
+                | (least_significant_bits & UUID_LSB_MASK)
+            )
+        )
+
+
+if hasattr(int, "from_bytes"):
+
+    def int_from_bytes(buffer):
+        return int.from_bytes(buffer, "big", signed=True)
+
+
+else:
+    # Compatibility with Python 2
+    def int_from_bytes(buffer):
+        buffer = bytearray(buffer)
+        if buffer[0] & 0x80:
+            neg = bytearray()
+            for c in buffer:
+                neg.append(c ^ 0xFF)
+            return -1 * int(binascii.hexlify(neg), 16) - 1
+        return int(binascii.hexlify(buffer), 16)
+
+
+try:
+    from datetime import timezone
+except ImportError:
+    from datetime import tzinfo, timedelta
+
+    # There is no tzinfo implementation(timezone) in the
+    # Python 2. Here we provide the bare minimum
+    # to the user.
+    class FixedOffsetTimezone(tzinfo):
+        __slots__ = ("_offset",)
+
+        def __init__(self, offset):
+            self._offset = offset
+
+        def utcoffset(self, dt):
+            return self._offset
+
+        def tzname(self, dt):
+            return None
+
+        def dst(self, dt):
+            return timedelta(0)
+
+    timezone = FixedOffsetTimezone
```

Comment:
does this minimal implementation have a drawback compared to python3?(I dont know maybe related to dst etc.)

Reply:
I don't think so, this is enough to store the timezone information we have